src/utils/endUserMessages.js

```diff
@@ -21,4 +21,6 @@ export default {
   NEW_CLUSTER_CREATED: 'Created a new cluster. Check the `Data Management` window.',
   SAVING_SAMPLE: 'Saving sample...',
   SAVING_PROJECT: 'Saving project...',
+  DETAILS_UPDATED: 'Account details updated successfully.',
```

Comment:
If any other change is requested, would you mind moving this where it belongs alphabetically? BTW, I would change that id to `ACCOUNT_DETAILS_UPDATED`.